Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md

```diff
@@ -6500,3 +6500,13 @@ Retrieve vulnerability details for a specific ID and host. Supported with the Cr
 * When encountering a 429 error code returned from Crowdstrike Falcon within Cortex XSOAR 8, the solution is to use an engine as explained in this [link](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Administrator-Guide/Engines).
 * In the different query parameters such as Endpoint Detections fetch query, Endpoint Incidents fetch query, etc. if you want to query for multiple values in the same field (for example, retrieving severity_name equal to Medium, High, and Critical) you need to use the following format: field:['value1','value2','value3'].
 * When encountering the error "400 - Reason: Bad Request: Invalid element in the request", ensure that your command arguments are correct. For example, if you're using the ID of a detection pre-Raptor (legacy API) in an integration configured to run with Raptor, this error will appear.
+* When encountering missing incidents on the ***fetch-incidents*** command, make sure that the 'Fetch Type' integration parameter includes the type of the missing incidents.
+Optional types are:
```

Comment:
IDP Detection is listed on line 29, but does not appear in this list.

Reply:
Thank you @ShirleyDenkberg - I'll fix that.